Lib/unittest/test/test_break.py

```diff
@@ -23,7 +24,9 @@ def tearDown(self):
         unittest.signals._results = weakref.WeakKeyDictionary()
         unittest.signals._interrupt_handler = None
 
-
+    # Tests both adaptive and specialized opcodes for proper
+    # CHECK_EVAL_BREAKER(). See bpo-46465 for an example bug.
+    @repeat_cpython_adaptative
```

Comment:
From what I've tested in https://bugs.python.org/issue46709 there several more test cases that require this decorator:
- testInterruptCaught
- testSecondInterrupt
- testTwoResults